vscode/src/non-stop/codelenses/items.ts

```diff
@@ -135,8 +135,9 @@ function getFormattingSkipLens(codeLensRange: vscode.Range, id: string): vscode.
 
 function getCancelLens(codeLensRange: vscode.Range, id: string): vscode.CodeLens {
     const lens = new vscode.CodeLens(codeLensRange)
+    const shortcut = process.platform === 'win32' ? 'Alt+C' : '⌥Z'
```

Comment:
```suggestion
    const shortcut = process.platform === 'darwin' ? '⌥Z' : 'Alt+C'
```

Doesn't Linux uses Alt as well? If not you can ignore this lol

Reply:
Yesss, good catch...